.editorconfig

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Should be at the project root, for editors to find it. The spec of all plugin is set that way: plugins will search from the opened file and go to parent dirs until it find a .editorconfig. See spec at http://editorconfig.org/

Most plugins does not support specifying an hardcoded path (like ```./dev```). For example the sublime editorconfig does not support it.